openpype/hosts/standalonepublisher/plugins/publish/collect_editorial_csv.py

```diff
@@ -0,0 +1,752 @@
+            },
+            "Package": {
+                "type": "text",
+                "default": "",
+                "required": False,
+                "validate": "^(.*)$"
+            },
+            "Filename": {
+                "type": "text",
+                "default": "",
+                "required": False,
+                "validate": "^(.*)$"
+            },
+            "Context": {
+                "type": "text",
+                "default": "",
+                "required": False,
+                "validate": "^(.*)$"
+            },
+            "Task": {
+                "type": "text",
+                "default": "",
+                "required": False,
+                "validate": "^(.*)$"
+            },
+            "Version": {
+                "type": "number",
+                "default": 1,
+                "required": True,
+                "validate": "^(\\d{1,3})$"
+            },
+            "Color": {
+                "type": "text",
+                "default": "",
+                "required": False,
+                "validate": "^(.*)$"
+            },
+            "Notes": {
+                "type": "text",
+                "default": "",
+                "required": False,
+                "validate": "^(.*)$"
+            },
+            "Intent": {
+                "type": "text",
+                "default": "",
+                "required": False,
+                "validate": "^(.*)$"
+            },
+            "Output": {
+                "type": "text",
+                "default": "",
+                "required": False,
+                "validate": "^(.*)$"
+            },
+            "Family": {
+                "type": "text",
+                "default": "",
+                "required": False,
+                "validate": "^(.*)$"
+            },
+            "Slate": {
+                "type": "bool",
+                "default": True,
+                "required": False,
+                "validate": "(true|false)"
+            },
+            "Tags": {
+                "type": "text",
+                "default": "",
+                "required": False,
+                "validate": "^(.*)$"
+            },
+            "Variant": {
+                "type": "text",
+                "default": "",
+                "required": False,
+                "validate": "^(.*)$"
+            },
+            "Start": {
+                "type": "number",
+                "default": 0,
+                "required": False,
+                "validate": "^(\\d{1,8})$|None"
+            },
+            "End": {
+                "type": "number",
+                "default": 0,
+                "required": False,
+                "validate": "^(\\d{1,8})$|None"
+            },
+            "Length": {
+                "type": "number",
+                "default": 0,
+                "required": False,
+                "validate": "^(\\d{1,8})$|None"
+            }
+        }
+
+    def config_representation_data(self):
```

Comment:
Can we get a docstring for this - I personally have no idea what the input key is and why it must adhere to certain extensions.

Is `preview` a tag? a family?

Reply:
https://github.com/ynput/OpenPype/pull/5795#discussion_r1440461055